pkg/volume/local_volume/local_volume.go

```diff
@@ -201,6 +202,34 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 
+	// If fsGroup is not nil, we need to make sure the requested fsGroup doesn't
+	// conflict with fsGroup already set for previous pod.
+	if fsGroup != nil {
```

Comment:
But the 1st pod will also run as root, so it should be able to read it. I think fsGroup only changes volume ownership, not user privilege?